src/calculator.js

```diff
@@ -1,47 +1,29 @@
-exports._check = () => {
-  // DRY up the codebase with this function
-  // First, move the duplicate error checking code here
-  // Then, invoke this function inside each of the others
-  // HINT: you can invoke this function with exports._check()
-};
-
-exports.add = (x, y) => {
+exports._check = (x,y) => {
```

Comment:
A space is required after &#39;,&#39;.

###### See http://eslint.org/docs/rules/comma-spacing for details about this rule